tools/check_installation.py

```diff
@@ -28,31 +28,19 @@
 SCIPY_DIR = os.path.join(ROOT_DIR, 'scipy')
 
 
+# Get the submodule paths so that we can ignore any tests in them or their submodules
+with open(os.path.join(ROOT_DIR, '.gitmodules')) as gitmodules:
+    data = gitmodules.read().split('\n')
+    submodule_paths = [datum.split(' = ')[1] for datum in data if \
```

Comment:
Note that I'm pretty sure we prefer to avoid the line continuation `\` like this--I haven't seen it discussed in a while, but as far as I can tell there's no need for it--works fine when I delete it locally?

I guess we don't need to block on it though.

Reply:
I didn't see anything in the docs about it, although it's possible that I missed it. Is there a relevant section of the docs concerning the use of the line continuation character you could point me to?

Reply:
It is a recommendation from [PEP 8](https://peps.python.org/pep-0008/), in the [Maximum Line Length](https://peps.python.org/pep-0008/#maximum-line-length) section:

> The preferred way of wrapping long lines is by using Python’s implied line continuation inside parentheses, brackets and braces. Long lines can be broken over multiple lines by wrapping expressions in parentheses. These should be used in preference to using a backslash for line continuation.

Reply:
Thank you!